### PR TITLE
Add source_path_prefix to Glide ServerFactory

### PR DESCRIPTION
**Pull request by matteovg7**
## Description

Add source_path_prefix to Glide ServerFactory create params

### CODE REVIEW

**Comment by haringsrob** [comment]
Hey @matteovg7 thank you for your contribution. Can you add this to the default config file + add some documentation as well.

**Comment by matteovg7** [comment]
Hi @haringsrob, I added the parameter to the default config file.
I don't know where to add the documentation but this is simply an additional parameter passed to the default glide configuration (https://glide.thephpleague.com/2.0/config/source-and-cache/)

**Comment by haringsrob** [comment]
hey @matteovg7 it's fine like this. I will check if additional docs are required. We can only merge this pr after the cla has been completed.

File: config/glide.php [hunk 1]
```diff
@@ -12,6 +12,7 @@
      */
 
     'source' => env('GLIDE_SOURCE', storage_path('app/public/' . config('twill.media_library.local_path'))),
+    'source_path_prefix' => env('GLIDE_SOURCE_PATH_PREFIX', ''),
```
Comment (ifox) [inline config/glide.php:15]:
```suggestion
    'source_path_prefix' => env('GLIDE_SOURCE_PATH_PREFIX', null),
```

File: src/Services/MediaLibrary/Glide.php [hunk 2]
```diff
@@ -69,6 +69,7 @@ public function __construct(Config $config, Application $app, Request $request)
         $this->server = ServerFactory::create([
             'response' => new LaravelResponseFactory($this->request),
             'source' => $this->config->get('twill.glide.source'),
+            'source_path_prefix' => $this->config->get('twill.glide.source_path_prefix', ''),
```
Comment (ifox) [inline src/Services/MediaLibrary/Glide.php:72]:
```suggestion
            'source_path_prefix' => $this->config->get('twill.glide.source_path_prefix'),
```

**Review by ifox — APPROVED** [review]

**Pull request merged**